src/pages/boundary/api-docs/[[...page]].tsx

```diff
@@ -21,15 +23,21 @@ const targetFile = {
 // The path to read from when running local preview in the context of the boundary repository
 const targetLocalFile = '../../internal/gen/controller.swagger.json'
 
-function ApiDocsView({ layoutProps, apiPageProps }) {
+import { SidebarSidecarLayoutProps } from 'layouts/sidebar-sidecar'
+import type { PageWithLayout } from 'pages/_app'
+
+function ApiDocsLayout(props: SidebarSidecarLayoutProps) {
+	return <SidebarSidecarLayout {...props} />
+}
```

Comment:
Q: Is this extra layer necessary?

Reply:
Not really, but it'd be nice to keep our "LayoutComponent" consumption consistent — i.e. use `<Page>.layout` rather than as a child.

Somewhat like this https://github.com/hashicorp/dev-portal/blob/06937b9b391539ab4d1970cde04d3f9019d37630/src/views/docs-view/index.tsx#L109-L110

Reply:
Agreed, sometimes it needs to be as a child though if we're passing props which aren't serializable to the layout.

This should work 💭 

```ts
ApiDocsView.layout = SidebarSidecarLayout
```

Reply:
Oh yea good call, that is actually what I wanted :homer: